client/components/gsuite/gsuite-price/index.jsx

```diff
@@ -15,60 +15,42 @@ import { getAnnualPrice, getMonthlyPrice } from 'lib/gsuite';
  */
 import './style.scss';
 
-const GSuitePrice = ( { cost, currencyCode, showMonthlyPrice } ) => {
+const GSuitePrice = ( { currencyCode, product } ) => {
 	const translate = useTranslate();
 
+	const cost = product?.cost ?? null;
```

Comment:
This works and the site no longer errors out :thumbsup:.

While this is not a blocker, I wish there was no flicker where `-` is displayed instead of the price.